src/CellData/CellQuadratures.jl

```diff
@@ -3,56 +3,93 @@
 
 """
 """
-struct CellQuadrature{DS} <: CellDatum
+struct CellQuadrature{DDS,IDS} <: CellDatum
   cell_quad::AbstractArray{<:Quadrature}
   cell_point::AbstractArray{<:AbstractArray{<:Point}}
   cell_weight::AbstractArray{<:AbstractArray{<:Real}}
   trian::Triangulation
-  domain_style::DS
+  data_domain_style::DDS
+  integration_domain_style::IDS
 end
```

Comment:
The default constructor of the struct has changed. Can you add an extra constructor with the old signature? and perhaps calling it in the tests? Just to be backwards compatible, I.e.,

```julia
function CellQuadrature(
  cell_quad::AbstractArray{<:Quadrature},
  cell_point::AbstractArray{<:AbstractArray{<:Point}},
  cell_weight::AbstractArray{<:AbstractArray{<:Real}},
  trian::Triangulation,
  data_domain_style::DomainStyle)
  CellQuadrature(cell_quad,cell_point,cell_weigth,trian,data_domain_style,PhysicalDomain())
end
```

Reply:
Yes! I will do that soon (I am in the middle of another task). 

I guess that the current set of tests do not fail because they are not calling to the (old) default constructtor, right?

Reply:
> Yes! I will do that soon (I am in the middle of another task).

Done!